spatialmath/geometry.go

```diff
@@ -43,16 +43,16 @@ type GeometryConfig struct {
 	Type GeometryType `json:"type"`
 
 	// parameters used for defining a box's rectangular cross section
-	X float64 `json:"x"`
-	Y float64 `json:"y"`
-	Z float64 `json:"z"`
+	X float64 `json:"x_mm"`
+	Y float64 `json:"y_mm"`
+	Z float64 `json:"z_mm"`
 
 	// parameter used for defining a sphere's radius'
-	R float64 `json:"r"`
+	R float64 `json:"r_mm"`
```

Comment:
@gvaradarajan should these fields have `mm` units or be unit-less like other spatial math distances? e.g. https://github.com/viamrobotics/rdk/pull/1351/files#r976980370

Reply:
I think so? @raybjork ? @biotinker ?

Reply:
okay, i think i see the difference - vectors should stay abstract, whereas this is an actual geometry, so better for it have units.